java/java-tests/testSrc/com/intellij/java/codeInspection/BulkFileAttributesReadFixJava11Test.java

```diff
@@ -5,10 +5,17 @@
 import com.intellij.codeInspection.BulkFileAttributesReadInspection;
 import com.intellij.codeInspection.LocalInspectionTool;
 import com.intellij.openapi.projectRoots.Sdk;
+import com.intellij.refactoring.JavaRefactoringSettings;
 import com.intellij.testFramework.IdeaTestUtil;
 import org.jetbrains.annotations.NotNull;
 
-public class BulkFileAttributesReadTest extends LightQuickFixParameterizedTestCase {
+public class BulkFileAttributesReadFixJava11Test extends LightQuickFixParameterizedTestCase {
+
+  @Override
+  protected void setUp() throws Exception {
+    super.setUp();
+    JavaRefactoringSettings.getInstance().INTRODUCE_LOCAL_CREATE_VAR_TYPE = true;
```

Comment:
If you change the settings in test, you should rollback them in teardown.

I still don't think that it's necessary to change the behavior of existing tests or move them somewhere. Could you please leave existing tests as is and add 2-3 new tests to test the new behavior? In general, try to make pull-requests as small as possible. All the changes should be justified. Longer pull-requests complicate review and take more time to process. Thank you.

Reply:
`BulkFileAttributesReadFixJava11Test` is used as inheritance by the other test unit classes